_examples/8-questions-to-fit-candidate-into-team.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 8 Questions, to fit candidate into team
+author: Ievgenii Vdovenko
+role:  interviewer
+overview: Aim of this questions is to build happy team. My believe is - if team is happy, it's more productive.
```

Comment:
The aim of these questions is to build a happy team. My belief is - if the team is happy, it's more productive.